roles/perf-cluster-tasks/tasks/main.yml

```diff
@@ -0,0 +1,4 @@
+- name: cordon master node
+  command: "kubectl cordon `echo $HOSTNAME`"
```

Comment:
at times, nodes will have a different name than hostname, can we cordon all the master nodes instead.

hints:

list master nodes:
```
kubectl get node --selector='node-role.kubernetes.io/master'
```
you can also use xargs here

Reply:
Changed to `kubectl get node --selector='node-role.kubernetes.io/master' --no-headers | awk '{print $1}' | xargs kubectl cordon`